kolibri/plugins/learn/assets/src/state/actions.js

```diff
@@ -706,7 +706,7 @@ function setAndSaveCurrentExamAttemptLog(store, contentId, itemId, currentAttemp
 }
 
 function closeExam(store) {
```

Comment:
This function is only ever used here https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/views/exam-page/index.vue#L153

Maybe stop returning a promise and redirect from here?

side note: how was that line ^ working when we weren't returning a promise? What was the `thenable`?